ProfileAPI/ProfileAPI/Configuration/ProfileDatabaseSettings.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ProfileAPI.Configuration
+{
+    public class ProfileDatabaseSettings : IProfileDatabaseSettings
+    {
+        public string ProfileCollectionName { get; set; }
+        public string ConnectionString { get; set; }
+        public string DatabaseName { get; set; }
+    }
+
+    public interface IProfileDatabaseSettings
```

Comment:
Does this need an interface? For simple model classes like this I prefer to have just a simple class.

Reply:
I followed the official Microsoft tutorial but I can change it

Reply:
Also see the ConferenceService for this. It's not really needed here :)